### DISCUSSION TITLE
目录显示错误

### DISCUSSION THREAD

**Discussion by nellojh** [MaLuns/hexo-theme-async#119] [Q&A 疑难杂症] [answered]
不知道为什么，打开网页时会显示querySelector错误，明明对应的id存在嘞
![image](https://github.com/MaLuns/hexo-theme-async/assets/162109084/dbdb9654-1d78-4de4-bd3d-4a001fc9d308)

**Comment by MaLuns** [comment]
这个 id 是数字开头的，document.querySelector 直接使用有问题，我修复下

**Comment by nellojh** [comment]
是呢是呢！我也刚刚发现，把md里的数字开头标题去掉就可以正常调用了。

**Comment by MaLuns** [comment, marked as answer]
更新了个小版本修复这个

**Reply by nellojh** [reply]
谢谢大大！😃

**Comment by nellojh** [comment]
大大，目前的方案会导致目录自动生成的数字标题和md中写的数字前缀重复。能不能特判一下，如果出现原文存在数字开头标题，就不再重复生成？😵
![image](https://github.com/MaLuns/hexo-theme-async/assets/162109084/314cce5c-b09c-4557-98bf-7e277c80e0f5)

**Reply by MaLuns** [reply]
可以在配置里关闭设置

**Reply by nellojh** [reply]
好滴好滴😃
